2021-04-01 20:03:59,782 INFO    MainThread:21124 [wandb_setup.py:_flush():69] setting env: {}
2021-04-01 20:03:59,782 INFO    MainThread:21124 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-01 20:03:59,782 INFO    MainThread:21124 [wandb_init.py:_log_setup():319] Logging user logs to C:\Users\neilg\PycharmProjects\DL\A2\partA\wandb\run-20210401_200359-1st7qxh4\logs\debug.log
2021-04-01 20:03:59,783 INFO    MainThread:21124 [wandb_init.py:_log_setup():320] Logging internal logs to C:\Users\neilg\PycharmProjects\DL\A2\partA\wandb\run-20210401_200359-1st7qxh4\logs\debug-internal.log
2021-04-01 20:03:59,783 INFO    MainThread:21124 [wandb_init.py:init():352] calling init triggers
2021-04-01 20:03:59,783 INFO    MainThread:21124 [wandb_init.py:init():359] wandb.init called with sweep_config: {}
config: {'data_aug': True, 'batch_size': 64, 'n_filters': 32, 'filter_org': 2, 'dropout': 0.2, 'batch_norm': True, 'activation': 'relu'}
2021-04-01 20:03:59,783 INFO    MainThread:21124 [wandb_init.py:init():401] starting backend
2021-04-01 20:03:59,783 INFO    MainThread:21124 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=spawn, using: spawn
2021-04-01 20:03:59,797 INFO    MainThread:21124 [backend.py:ensure_launched():123] starting backend process...
2021-04-01 20:03:59,922 INFO    MainThread:21124 [backend.py:ensure_launched():128] started backend process with pid: 15224
2021-04-01 20:03:59,922 INFO    MainThread:21124 [wandb_init.py:init():406] backend started and connected
2021-04-01 20:03:59,923 INFO    MainThread:21124 [wandb_init.py:init():446] updated telemetry
2021-04-01 20:03:59,924 INFO    MainThread:21124 [wandb_init.py:init():465] communicating current version
2021-04-01 20:04:01,633 INFO    MainThread:21124 [wandb_init.py:init():470] got version response upgrade_message: "wandb version 0.10.24 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-04-01 20:04:01,633 INFO    MainThread:21124 [wandb_init.py:init():478] communicating run to backend with 30 second timeout
2021-04-01 20:04:02,065 INFO    MainThread:21124 [wandb_init.py:init():503] starting run threads in backend
2021-04-01 20:04:04,645 INFO    MainThread:21124 [wandb_run.py:_console_start():1422] atexit reg
2021-04-01 20:04:04,646 INFO    MainThread:21124 [wandb_run.py:_redirect():1285] redirect: SettingsConsole.WRAP
2021-04-01 20:04:04,647 INFO    MainThread:21124 [wandb_run.py:_redirect():1320] Wrapping output streams.
2021-04-01 20:04:04,647 INFO    MainThread:21124 [wandb_run.py:_redirect():1336] Redirects installed.
2021-04-01 20:04:04,647 INFO    MainThread:21124 [wandb_init.py:init():527] run started, returning control to user process
2021-04-01 20:04:06,395 INFO    MainThread:21124 [wandb_run.py:_atexit_cleanup():1392] got exitcode: 1
2021-04-01 20:04:06,395 INFO    MainThread:21124 [wandb_run.py:_restore():1364] restore
2021-04-01 20:04:06,757 INFO    MainThread:21124 [wandb_run.py:_wait_for_finish():1515] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 761
}

2021-04-01 20:04:08,770 INFO    MainThread:21124 [wandb_run.py:_wait_for_finish():1515] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4211
  total_bytes: 4211
}

2021-04-01 20:04:09,987 INFO    MainThread:21124 [wandb_run.py:_show_files():1737] logging synced files
